test/browser/call-device.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+
+import { PeerUser } from '../peer-user.js';
+import { expectEvents, updateRemoteVideos, sleep, logEvents } from '../helper.js';
+import config from './config.js'
+
+const assert = chai.assert;
+let client;
+let peerUser;
+let call;
+let user;
+let stream;
+describe('Call Devices', async function() {
+    this.timeout(300000);
+
+    before(async function() {
+        Circuit.logger.setLevel(Circuit.Enums.LogLevel.Error);
+        client = new Circuit.Client(config.config);
+        const res = await Promise.all([PeerUser.create(), client.logon(config.credentials)]);
+        peerUser = res[0];
+        user = res[1];
+        const conversation = await client.createGroupConversation([peerUser.userId], 'SDK Test: Conference Call');
```

Comment:
is "SDK Test: Conference Call" right?

Reply:
No, i will make each conversation's title reflect the test.